### ISSUE TITLE
Fix sandbox testnet faucet (for Stacking amounts and regular ones?)

### ISSUE THREAD

**Issue by markmhx** [hirosystems/explorer#1072]
It seems to always say "Failed to fetch" 😢 

<img width="494" alt="Screenshot 2023-03-27 at 09 56 43" src="https://user-images.githubusercontent.com/28991/227877577-1a7e207e-704b-4f0c-99b2-a0937c670904.png">

**Comment by 314159265359879** [comment]
I think this title should be changed because the faucet is broken for any amount. the Console is showing a CORS error. 

https://user-images.githubusercontent.com/33360391/230711352-f2b81517-b8cb-4976-9278-d696089f5ef9.mp4

![image](https://user-images.githubusercontent.com/33360391/230711375-e3a8f535-d5d0-4721-8a0e-7704246f791c.png)

**Comment by 314159265359879** [comment]
Workaround: Use the Hiro Desktop wallet for testnet, that still has a faucet button that works (for small amounts). 
https://github.com/hirosystems/desktop-wallet/releases/tag/v4.8.0

**Comment by andresgalante** [comment]
We tested it and it's working, Thanks @CharlieC3. I'll close the issue

**Comment by markmhx** [comment]
I just tried again and the request for regular amounts worked but then my subsequent request for Stacking amounts failed:

<img width="1259" alt="Screenshot 2023-04-11 at 12 58 45" src="https://user-images.githubusercontent.com/28991/231140375-55b66ae4-4542-401d-9689-f1eea3d2ee61.png">

**Comment by CharlieC3** [comment]
@markmhx A 429 status code means you're being rate-limited by the faucet. You can make a faucet request once every minute. So if you wait long enough, you should be able to make another request. I think this is working as intended.

**Comment by CharlieC3** [comment]
Ideally if the Explorer received a `429` on this page, it would be a better UX if it could display a message saying you're being rate-limited, and to wait a minute before sending another request.
cc @He1DAr

**Comment by markmhx** [comment]
You're right, that was the issue – I tried for Stacking amounts again today and it worked 🥳 

<img width="454" alt="Screenshot 2023-04-12 at 12 46 02" src="https://user-images.githubusercontent.com/28991/231435143-9f097a42-392a-4c08-983b-9c72f27659a8.png">

**Comment by andresgalante** [comment]
@shamim-io would this be something you are willing to take?

**Comment by shamim-io** [comment]
@andresgalante I think this section is visible only in `testnet` but not in `mainnet`. 
When I am viewing it from the `mainnet` I can't reproduce the section. The screenshot is as under. 
<img width="1144" alt="Screenshot 2023-05-31 at 12 31 44 AM" src="https://github.com/hirosystems/explorer/assets/29030948/244ae8ab-3d7f-4076-960a-12e68a12d16f">

If I am right. Can you please provide me a valid URL to create a test network ? 
Thanks!

**Comment by andresgalante** [comment]
@shamim-io That's right, the faucet only works on testnet since it proves "free" testnet STX 😄

**Comment by shamim-io** [comment]
Awesome! Can you please provide me an URL to create a test network? @andresgalante

**Comment by He1DAr** [comment]
@shamim-io you just need to select the testnet from the menu
<img width="364" alt="image" src="https://github.com/hirosystems/explorer/assets/97111756/603f09f1-0591-414a-8376-34dd65df2f54">

**Comment by heidar-stacks** [comment]
already fixed
